lib/i18n/hygiene/variable_checker.rb

```diff
@@ -4,16 +4,14 @@ module Hygiene
     # as defined in :en contains an interpolation variable, the value for that key as defined
     # in any other locale must have a matching variable name.
     class VariableChecker
-
-      NON_ENGLISH_LOCALES_TO_CHECK = [ :fr ]
-
-      def initialize(key, i18n_wrapper)
+      def initialize(key, i18n_wrapper, locales = [])
         @key = key
         @i18n_wrapper = i18n_wrapper
+        @locales = locales
```

Comment:
Potentially, but should we do that in this branch? This is PR mostly concerned with setting up a pattern to allow configuration to unblock the other cards we've got queued.